otx/algorithms/classification/adapters/dor/utils/monitors.py

```diff
@@ -0,0 +1,100 @@
+"""Monitors for dor tasks."""
```

Comment:
This doesn't seem to import deep-reid. If so, does it need to be in the adapter?

Reply:
I put all methods that only dor uses into adapter/dor/utils.